proto/google/protobuf/any.proto

```diff
@@ -0,0 +1,154 @@
+// Protocol Buffers - Google's data interchange format
```

Comment:
Looks like `any.proto`, `contract.proto` and `tron.proto` is not used anywhere in communication. If this is device specific then it should not be present in `cypherock-common`.

Reply:
updated